fhir-examples/pom.xml

```diff
@@ -40,63 +40,63 @@
             <id>fhir-javadocs</id>
             <build>
                 <plugins>
-                    <!-- limit the javadoc plugin to jenkins only -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-site-plugin</artifactId>
+                        <version>3.8.2</version>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-project-info-reports-plugin</artifactId>
+                        <version>2.9</version>
+                    </plugin>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
-                        <version>3.1.1</version>
+                        <version>3.0.1</version>
```

Comment:
why the downgrade?

Reply:
good catch - thanks... it was part of my debugging process... which was extensive :)